name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/holdings/HoldingsPieChartSWT.java

```diff
@@ -93,15 +114,17 @@ public Control createControl(Composite parent)
         });
 
         // Listen on mouse clicks to update information pane
-        ((Composite) chart.getPlotArea()).addListener(SWT.MouseUp,
+        if (financeView != null)
+        {
+            ((Composite) chart.getPlotArea()).addListener(SWT.MouseUp,
                         event -> chart.getNodeAt(event.x, event.y).ifPresent(node -> {
                             NodeData nodeData = id2nodeData.get(node.getId());
                             if (nodeData != null)
                                 financeView.setInformationPaneInput(nodeData.position.getInvestmentVehicle());
                         }));
 
-        updateChart();
-
+         updateChart();
```

Comment:
updateChart needs a non null Snapshot, but at this stage, Snapshot is not yet instantiated when the chart is created from the new information pane, so it is hidden in the finance view 'if' block (=this is the existing Statement of Assets/Holdings chart). I did not notice any issue, but to be noted.